apps/web/pages/event-types/index.tsx

```diff
@@ -739,9 +736,9 @@ const EventTypeListHeading = ({
         )}
         {profile?.slug && (
           <Link href={`${CAL_URL}/${profile.slug}`} className="text-subtle block text-xs">
-            {orgSlug
-              ? `${orgSlug}.${subdomainSuffix()}/${profile.slug}`
-              : `${CAL_URL?.replace("https://", "")}/${profile.slug}`}
+            {orgBranding
+              ? `${orgBranding.fullDomain.replace("https://", "").replace("http://", "")}${profile.slug}`
```

Comment:
nit: we should have a `orgBranding.domainOnly` or something that takes care of it already.